src/main/kotlin/no/nav/helse/spenn/oppdrag/OppdragSkjemaConstants.kt

```diff
@@ -35,6 +33,15 @@ class OppdragSkjemaConstants {
 
 }
 
+enum class FagOmraadekode(val kode: String) {
+    SP("SP"), //bruker er mottaker
+    SPREF("SPREF") // refusjon til arbeidsgiver
+}
+
+enum class KlassifiseringsKode(val kode: String) {
+    SPREFAG_IOP("SPREFAG-IOP")
```

Comment:
istedenfor kommentar så burde enum-navnet reflektert hva det er

Reply:
fikk beskjed å bruke denne koden siden det er refusjon til arbeidsgiver.. men vet ikke helt eksakt hva det står for, og fant ingen dok for det heller :)